packages/ndk/example/basic_test.dart

```diff
@@ -29,7 +29,7 @@ void main() async {
             '3bf0c63fcb93463407af97a5e5ee64fa883d107ef9e558472c4eb9aaaefa459d'
           ],
           // Query for text note events (kind 1)
-          kinds: [Nip01Event.TEXT_NODE_KIND],
+          kinds: [Nip01Event.kKind],
```

Comment:
what is the reason for kKind? Dont quite understand

If its for generic kind getter then we schould enforce it via abstract class, but not sure if I understand

Reply:
should be Nip01Event.kTextNodeKind

Reply:
@franzaps can you change that? 
We can merge after that.

Reply:
In a class you would use one member to refer to something, i.e. `kind`, and then override in each subclass. You then know that no matter where you are, you call `kind` and you get the correct value.

I understand that kinds in this context need to be accessed without instantiating a class - so statics are a way of doing it. Calling them exactly the same everywhere (`kKind`) was my attempt to normalize it. But then we run into the issue of clashing naming in subclasses with statics. I'll revert to `kTextNodeKind` for this reason. I prefer the approach I took in purplebase for this (a map)